plugins/woocommerce/tests/metrics/specs/editor.spec.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable playwright/no-conditional-in-test, playwright/expect-expect */
+
+/**
+ * WordPress dependencies
+ */
+import { test, Metrics } from '@wordpress/e2e-test-utils-playwright';
+
+/**
+ * Internal dependencies
+ */
+import { PerfUtils } from '../fixtures';
+import { median } from '../utils';
+
+// See https://github.com/WordPress/gutenberg/issues/51383#issuecomment-1613460429
+const BROWSER_IDLE_WAIT = 1000;
+
+const results = {};
+
+async function editPost( admin, page, postId ) {
+	const query = new URLSearchParams();
+	query.set( 'post', String( postId ) );
+	query.set( 'action', 'edit' );
+
+	await admin.visitAdminPage( 'post.php', query.toString() );
+	await setPreferences( page, 'core/edit-post', {
+		welcomeGuide: false,
+		fullscreenMode: false,
+	} );
+}
+
+async function setPreferences( page, context, preferences ) {
+	await page.waitForFunction( () => window?.wp?.data );
+
+	await page.evaluate(
+		async ( props ) => {
+			for ( const [ key, value ] of Object.entries(
+				props.preferences
+			) ) {
+				await window.wp.data
+					.dispatch( 'core/preferences' )
+					.set( props.context, key, value );
+			}
+		},
+		{ context, preferences }
+	);
+}
```

Comment:
These two utilities are actually present in the latest version of `@wordpress/e2e-test-utils-playwright`. We could just do `admin.editPost( id )` but I copied them over for now because of the pinned dependency version.